doc/stdenv/stdenv.chapter.md

```diff
@@ -797,15 +797,15 @@ A shell array containing additional arguments passed to the configure script. Yo
 
 ##### `dontAddPrefix` {#var-stdenv-dontAddPrefix}
 
-By default, the flag `--prefix=$prefix` is added to the configure flags. If this is undesirable, set this variable to true.
+By default, the flag set by a combination of [`prefixKey`](#var-stdenv-prefixKey) and [`prefix`](#var-stdenv-prefix) is added to the configure flags. If this is undesirable, set this variable to true.
```

Comment:
```suggestion
By default, `./configure` is passed the concatenation of [`prefixKey`](#var-stdenv-prefixKey) and [`prefix`](#var-stdenv-prefix) on the command line. Disable this by setting `dontAddPrefix` to `true`.
```

Otherwise it's not clear what that "combination" is.